docs/schema/shutter_plan.md

```diff
@@ -0,0 +1,15 @@
+# Shutter Plan
+
+Keep the shutter open along the specified axes.
+
+::: useq.ShutterOpenAxes
+    options:
+        show_signature: true
+        show_signature_annotations: true
+        members: []
+
+::: useq.ShutterOpenAxes.should_keep_open
```

Comment:
i don't think we need to document the python implementation stuff here.  Just the schema